src/core/lib/security/credentials/tls/grpc_tls_crl_provider.cc

```diff
@@ -22,6 +22,7 @@
 
 #include <limits.h>
 
+// IWYU pragma: no_include <ratio>
```

Comment:
Why would this include be a problem?

Reply:
IWYU wants to include it, but it's in `cpp_banned_constructs.sh`

https://github.com/grpc/grpc/blob/2fdcba26dde3d8e9bbac2fb5484d45a999e8a03b/tools/run_tests/sanity/cpp_banned_constructs.sh#L37